### DISCUSSION TITLE
I have a problem when trying to go to another component with the onSubmit of the autoform

### DISCUSSION THREAD

**Discussion by emilio19951** [vazco/uniforms#1155] [Q&A] [answered]
I have a handleSubmit function that I call in the AutoForm obSubmit in which I want to see the saved information of the form and send it to another route but if I do a return <Link href="/hello" /> it does not take me to the page, I would like to know If someone could guide me on that part.

Thank you
![image](https://user-images.githubusercontent.com/78379092/184255335-1444c3fd-83ac-46eb-81d3-9c87deb6feca.png)

![image](https://user-images.githubusercontent.com/78379092/184255285-270187d3-fc19-4ffa-b7f5-95d303ad8f47.png)

**Comment by radekmie** [comment, marked as answer]
Hey @emilio19951. As the `onSubmit` is triggered on an event, it has nothing to do with the rendering. I don't know what kind of routing library you are using, but there's most likely some sort of `redirect(path)` function - you should use it.
